src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,9 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {},
+});
+
+export default categoriesSlice.reducer;
```

Comment:
Please consider :-

* Creating 1 reducer, like `checkStatus`.
* And Export `checkStatus` as an action.
As you did it in booksSlice.js.

## MORE INFO:-

You can achieve the above mentioned by considering the following code:-

```
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

Reply:
thank you for your review I will make the changes you mentioned.

Reply:
✅ I appreciate your response. Next time consider tagging me when writing a comment @Santosh-Konappanavar